documentation/man/features/ssh.rst

```diff
@@ -29,6 +31,14 @@ OPTIONS
 -r <USER@IP:PORT | CONFIG_HOST_NAME>, \--remote <USER@IP:PORT | CONFIG_HOST_NAME>:
   Uses parameter to ssh into the target machine
 
+\--send <from-local-path> \--to <to-remote-path>:
+  Transfers a file or directory located in <local-path> to the remote destination
+  specified in <remote-path>.
```

Comment:
Since `--to` is optional, explain where the file will be added if the user does not provide this option.